src/redux/selectors/contacts.selectors.js

```diff
@@ -0,0 +1,12 @@
+export const selectContactsAndTable = ({ Contact, Table }) => ({
+  contacts: Contact.contacts,
+  loading: Contact.ContactLoading,
+  currentPage: Table.pagination.currentPage,
+  rowsPerPage: Table.pagination.rowsPerPage,
+  itemsCount: Table.itemsCount
+});
+
+export const selectContact = ({ Contact }) => ({
+  loading: Contact.contactsLoading,
+  contact: Contact.contact
+});
```

Comment:
it makes sense to move this selector to contacts.reducer

Reply:
fixed all